var/spack/repos/builtin/packages/pacifica-cli/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PacificaCli(PythonPackage):
+    """Python CLI for Pacifica Core Services"""
+
+    homepage = "https://github.com/pacifica/pacifica-cli/"
+    pypi = "pacifica-cli/pacifica-cli-0.5.2.tar.gz"
+
+    version('0.5.2', sha256='fee5fa8ac38ffec2e9199bff23afbbae697c4f7554f13e340104f8b20a62843f')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools-scm', type='build')
+    depends_on('py-jsonschema', type=('build', 'run'))
+    depends_on('py-pacifica-uploader@0.3.1:', type=('build', 'run'))
+    depends_on('py-pacifica-downloader@0.4.1:', type=('build', 'run'))
+    depends_on('py-pacifica-namespace', type=('build', 'run'))
+    depends_on('py-pager', type=('build', 'run'))
+    depends_on('python', type=('build', 'run'))
```

Comment:
You can remove these lines now as they are automatically added by the base class.

Reply:
i removed these and re-install pacifica-cli and it does not pull in those dependencies for some reason.  any idea why?

Reply:
By "these lines" I mean just the `depends_on('python', type=('build', 'run'))` lines from your packages. This is implied by the fact that they subclass `PythonPackage`.

Reply:
ahh.  ok
removed.